test/Microsoft.Azure.ServiceBus.UnitTests/SenderReceiverClientTestBase.cs

```diff
@@ -124,7 +124,7 @@ public abstract class SenderReceiverClientTestBase
             TestUtility.Log("Sleeping 10 seconds...");
             await Task.Delay(TimeSpan.FromSeconds(10));
 
-            DateTime lockedUntilUtcTime = await messageReceiver.RenewLockAsync(receivedMessages.First().LockToken);
+            var lockedUntilUtcTime = await messageReceiver.RenewLockAsync(receivedMessages.First().LockToken);
```

Comment:
Fair point. Agree to take it case by case.
R# can be harmful sometimes. That's why it's just a tool, not a developer replacement 😆